js/fileUtils.ts

```diff
@@ -34,16 +26,17 @@ export function genMediaTags(
   if (typeof file === "string" && !/^[a-z]+:\/\//i.test(file)) {
     file = `${location.protocol}//${location.host}${location.pathname}${file}`;
   }
```

Comment:
I think we can get rid of this workaround, right? If not, let's update the comment.